tests/experimental/datasources/test_integration_sql_datasource.py

```diff
@@ -22,7 +24,11 @@ def test_run_checkpoint_and_data_doc(empty_data_context, include_rendered_conten
     """
     db_file = file_relative_path(
         __file__,
-        "../../test_sets/taxi_yellow_tripdata_samples/sqlite/yellow_tripdata.db",
+        os.path.join(
```

Comment:
If you take the `os` dependency to make this os agnostic, I think you should break up the whole path since the string `test_sets/taxi_yellow_tripdata_samples/sqlite/yellow_tripdata.db` has `/`s in it.

Reply:
Done -- committed.  Nice catch as I did it in the new test!  Thanks!

Reply:
Or use `pathlib`
https://treyhunner.com/2019/01/no-really-pathlib-is-great/#Normalizing_file_paths_shouldn%E2%80%99t_be_your_concern

Reply:
@Kilo59 I used `os` for this one.